tests/timetables/test_trigger_timetable.py

```diff
@@ -47,7 +47,7 @@
     [
         pytest.param(
             None,
-            YESTERDAY + DELTA_FROM_MIDNIGHT,
```

Comment:
Does this case not pass? This should be kept.

Reply:
If I understand correctly it refers to the next start time after it is enabled for the first time, shouldn't it be **after** current time (always referring to the documentation and the way cronjobs work)?